app/src/main/java/com/cmput301f20t21/bookfriends/ui/scanner/ScannerBaseActivity.java

```diff
@@ -27,14 +28,15 @@
  * the base Activity to be extended by any that needs the scanning functionality
  */
 public class ScannerBaseActivity extends AppCompatActivity {
+    public static final String ISBN_KEY = "ISBN_KEY";
```

Comment:
I think we are using the convention that all keys should have the `com.cmput301f20t21.bookfriends` prefix

other than that it lgtm